packages/types/src/primitive/sszTypes.ts

```diff
@@ -17,7 +18,7 @@ export const Bytes96 = new ByteVectorType({length: 96});
 export const Uint8 = byteType;
 export const Uint16 = new NumberUintType({byteLength: 2});
 export const Uint32 = number32Type;
-export const Number64 = new NumberUintType({byteLength: 8});
+export const Number64 = new Number64UintType();
```

Comment:
Should we rename to `Number64Uint` to prevent issues in the future if we need a true `Number64`?

Reply:
yeah, maybe Uint64 since we have Uint8, Uint16, Uint32...